app/models/application_record.rb

```diff
@@ -92,7 +92,13 @@ def move_to_dashboard
 
   def self.move_to_dashboard
     records = where('date_for_later <= ?', Time.zone.today)
-    records.each(&:move_to_dashboard)
+    records.each do |record|
+      record.move_to_dashboard
+      record.position.subscriptions.each do |sub|
```

Comment:
No — I take that back! It would be *best* not to spam someone with 7 emails if 7 applications are reactivated for the same position.

Here's what we should do:

1. For all applications which will be reactivated, find the email addresses of the subscriptions.
2. Create a hash mapping each email address to the reactivated applications about which the person needs to be notified.
3. Send a single email which breaks down how many applications were reactivated per position.

Note that a single reactivated application could be sent to multiple people, one of whom might also have a subscription to another position — so there is a non-trivial amount of data processing to do here.